test/PickSpec.hs

```diff
@@ -27,7 +27,8 @@ validBoundsPickConfig = do
     "Dnf" -> FormulaDnf <$> validBoundsNormalFormConfig
     _ -> FormulaArbitrary <$> validBoundsSynTreeConfig' False `suchThat` \SynTreeConfig{..} ->
             amountOfOptions <= 4*2^ length availableAtoms &&
-            maxNodes <= 40
+            maxNodes <= 40 &&
+            minAmountOfUniqueAtoms >= 3
```

Comment:
That's making a change in the tests, but not guaranteeing that user supplied configs satisfy this:
> Da wäre ich noch eher dafür, bei `Pick` einfach zu verlangen, dass `availableAtoms` mindestens dreielementig ist. Denn Wahrheitstafeln mit nur zwei atomaren Formeln sind sowieso so trivial, dass sie wahrscheinlich kein interessantes Übungsziel sind.

Moreover, `minAmountOfUniqueAtoms >= 3` is something else than `length availableAtoms >= 3`.